docs/deep-dives/mobile/first-run-experiments.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: mobile-first-run-experiments
+title: First Run Experiments
+slug: /mobile-first-run-experiments
+---
+
+First run experiments are incredibly important to the experimentation ecosystem, but their functionality comes with some key differences from standard experiments. It is _highly_ advised that experiment owners read through this documentation before writing first run experiments.
+
+:::note
+
+The concept of first run experiments is currently only supported on mobile applications, such as Firefox for Android/iOS and Focus.
+
+:::
+
+## What is a first run experiment?
+
+A first run experiment is one that needs to enroll clients on their first launch of the application. All first run experiments bundled with a particular app version would be evaluated for enrollment on the very first run of the application after it is installed.
+
+An experiment can be marked as a first run experiment by checking the "First Run Experiment" checkbox while editing the experiment in the audience page.
+
+<img title="First run checkbox" alt="The first run experiment checkbox is shown on the audience page of an experiment being edited"
```

Comment:
Alt text=" 😻 "

Reply:
I try my best to make sure accessibility is considered and covered where possible 😊